README.md

```diff
@@ -1,60 +1,57 @@
 # XMTP-iOS
 
-This SDK is in Developer Preview status. Software in this status:
+![Test](https://github.com/xmtp/xmtp-ios/actions/workflows/test.yml/badge.svg) ![Lint](https://github.com/xmtp/xmtp-ios/actions/workflows/lint.yml/badge.svg) ![Status](https://camo.githubusercontent.com/5bb5892781bbf711c7fe5eba3328e9e15a767de87c587d3fb65f2fd7e1f4ae72/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f50726f6a6563745f5374617475732d446576656c6f7065725f507265766965772d726564)
+
+`xmtp-ios` provides a Swift implementation of an XMTP message API client for use with iOS apps.
+
+Use `xmtp-ios` to build with XMTP to send messages between blockchain accounts, including DMs, notifications, announcements, and more.
+
+This SDK is in **Developer Preview** status. We do **not** recommend using Developer Preview software in production apps.
+
+Software in this status:
 
 - Is not formally supported
 - Will change without warning
 - May not be backward compatible
-- We do NOT recommend using Developer Preview software in production apps.
+- Has not undergone a formal security audit
 
 Specifically, this SDK is missing this functionality:
 
-- Specifying `apiUrl`, `keyStoreType`, `codecs`, `maxContentSize` and `appVersion` when creating a `Client`
+- Specifying `apiUrl`, `keyStoreType`, `codecs`, `maxContentSize`, and `appVersion` when creating a `Client`
 - Content types other than text
 - Streaming all messages from all conversations
 - Message content compression
 
 Follow along in the [tracking issue](https://github.com/xmtp/xmtp-ios/issues/7) for updates.
 
-![Test](https://github.com/xmtp/xmtp-ios/actions/workflows/test.yml/badge.svg)
-![Lint](https://github.com/xmtp/xmtp-ios/actions/workflows/lint.yml/badge.svg)
-![Status](https://camo.githubusercontent.com/5bb5892781bbf711c7fe5eba3328e9e15a767de87c587d3fb65f2fd7e1f4ae72/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f50726f6a6563745f5374617475732d446576656c6f7065725f507265766965772d726564)
+To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
 
 ![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
 
-**XMTP client SDK for iOS applications**
+## Example app
```

Comment:
I thought this heading text might capture dev attention - but is it an accurate description of what's in the example project? =) If not, we should change it to "Example project" in the heading and the link text below.

Reply:
I guess what do we consider an "app" vs a "project?" You can install and run the example so by that metric it's an app, if not a very fully featured one.

Reply:
Thank you, Pat! I think we should go with "app" then. =) Perhaps we can set some expectations about the robustness of the app by making the edit below?

Reply:
Yep, works for me!